push.sh

```diff
@@ -1,44 +1,44 @@
 #!/bin/bash -e
 
 function normalize_repo_name() {
-  local repoName="$1"
-  if [ "$repoName" != "" ] && [ "${repoName: -1}" != "/" ]; then
-    repoName="$repoName/"
-  fi
-  echo -n "$repoName"
+	local repoName="$1"
```

Comment:
Looks like your editor might still be setup for GoLang.  We typically use one tab (two spaces) for indenting in Bash.

Reply:
we're using the shfmt with its default setting of one tab indent, adapted config to use two spaces